src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,4 +66,8 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Remember to add header comments as the overriden `isMutating()` behaves differently from the superclass method.